### PR TITLE
Bzp/geoauto 1911 sagemaker

### PR DESCRIPTION
**Pull request by bzporter**
Dataloaders for airsim and lidar. Fix up dataloaders to work with new organization of dataset. Add tests for dataloader listfiles

### CODE REVIEW

File: dataloader/MiddleburyLoader.py [hunk 1]
```diff
@@ -62,9 +66,9 @@ def __getitem__(self, index):
         max_w = 3072//4
 
         # photometric unsymmetric-augmentation
-        random_brightness = np.random.uniform(self.rand_bright[0], self.rand_bright[1],2)
-        random_gamma = np.random.uniform(0.8, 1.2,2)
-        random_contrast = np.random.uniform(0.8, 1.2,2)
+        random_brightness = np.random.uniform(self.rand_bright[0], self.rand_bright[1], 2)
```
Comment (rmjustus) [inline dataloader/MiddleburyLoader.py:69]: I think we should definitely add in some jpeg augmentation here too

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+opencv-python-headless==4.4.0.44
+matplotlib==3.3.2
+numpy==1.19.1
+scipy==1.5.2
+tensorflow==1.15.3
```
Comment (rmjustus) [inline requirements.txt:5]: What are we using tensorflow for?

File: utils/s3_sync.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    os.makedirs(dataset_dir, exist_ok=True)
+    command = 'aws s3 sync "s3://autogpe-datasets/airsim_dataset" "{}/airsim" --quiet'.format(dataset_dir)
+    print(command)
+    os.system(command)
+    print('===== Finished airsim sync =====')
+
+
+def sync_lidar_dataset(dataset_dir):
+    os.makedirs(dataset_dir, exist_ok=True)
+    command = 'aws s3 sync "s3://autogpe-datasets/lidar-hdsm-dataset" "{}/lidar" --quiet'.format(dataset_dir)
```
Comment (rmjustus) [inline utils/s3_sync.py:22]: I think we may want to launch like a thread or process per pool and sync the top level directories for speed

**Review by rmjustus — APPROVED** [review]
Looks good.  I think tomorrow I am going to check out the branch and do some more work on the file syncing from s3

File: requirements.txt (hunk 2 above)
Comment (bzporter) [reply requirements.txt:5]: Tensorboard logging is what it is used for

**Pull request merged**